.github/workflows/build.yml

```diff
@@ -141,3 +141,31 @@ jobs:
         run: cd .repo && npx projen package:python
       - name: Collect python Artifact
         run: mv .repo/dist dist
+  package-dotnet:
+    needs: build
+    runs-on: ubuntu-latest
+    permissions: {}
+    if: "! needs.build.outputs.self_mutation_happened"
+    steps:
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8
+        with:
+          node-version: 20.x
+      - uses: actions/setup-dotnet@v3
```

Comment:
upgrade this to commit hash

Reply:
done in [4cc87aa](https://github.com/awslabs/generative-ai-cdk-constructs/pull/529/commits/4cc87aac1d2cb5d984ff82ee6259b592b3e56d5e) by @scottschreckengaust